### ISSUE TITLE
Staging failure with memcalc v3 branch: cannot load such file -- zip

### ISSUE THREAD

**Issue by glyn** [cloudfoundry/java-buildpack#373]
When a WAR file is pushed using branch 135936599-memory-calc-v3, the following error occurs:
```
Staging...
/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- zip (LoadError)
	from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /tmp/buildpacks/061a0e2eaff18cabd0384bda82d48da8/lib/java_buildpack/jre/open_jdk_like_memory_calculator.rb:24:in `<top (required)>'
  ... etc. ...
Failed to compile droplet
Exit status 223
Staging failed: Exited with status 223
```

**Comment by glyn** [comment]
The problem reproduces even after I updated my fork to use the correct commit |90f39e3eeed24bea46e1877a).

**Comment by nebhale** [comment]
Resolved by c45b7f3
